kat/src/lib.rs

```diff
@@ -25,8 +26,45 @@ mod sha384acc_kat;
 pub use caliptra_drivers::{CaliptraError, CaliptraResult};
 pub use ecc384_kat::Ecc384Kat;
 pub use hmac384_kat::Hmac384Kat;
+pub use kats_env::KatsEnv;
 pub use lms_kat::LmsKat;
 pub use sha1_kat::Sha1Kat;
 pub use sha256_kat::Sha256Kat;
 pub use sha384_kat::Sha384Kat;
 pub use sha384acc_kat::Sha384AccKat;
+
+use caliptra_drivers::cprintln;
+
+/// Execute Known Answer Tests
+///
+/// # Arguments
+///
+/// * `env` - ROM Environment
+pub fn execute_kat(env: &mut KatsEnv) -> CaliptraResult<()> {
+    cprintln!("[kat] ++");
+
+    cprintln!("[kat] sha1");
+    Sha1Kat::default().execute(env.sha1)?;
+
+    cprintln!("[kat] SHA2-256");
+    Sha256Kat::default().execute(env.sha256)?;
+
+    cprintln!("[kat] SHA2-384");
+    Sha384Kat::default().execute(env.sha384)?;
+
+    cprintln!("[kat] SHA2-384-ACC");
+    Sha384AccKat::default().execute(env.sha384_acc)?;
+
+    cprintln!("[kat] ECC-384");
+    Ecc384Kat::default().execute(env.ecc384, env.trng)?;
+
+    cprintln!("[kat] HMAC-384");
+    Hmac384Kat::default().execute(env.hmac384, env.trng)?;
+
+    cprintln!("[kat] LMS");
+    LmsKat::default().execute(env.sha256, env.lms)?;
```

Comment:
Do we need to execute the LMS KAT? We don't use LMS except for to do the ImageVerification (which is also just to satisfy FIPS)

Reply:
I see your point. This is  a question for @JohnTraverAmd.

Reply:
LMS is part of the firmware authentication, and fips requires us to reauthenticate. I assume since we do an LMS kat we're actually getting it certified, and thus it's required.

Reply:
Ok, so we're saying that because we use LMS for firmware auth, we also need to run the LMS KAT. That's fine, just double checking.